editoast/src/core/v2/pathfinding.rs

```diff
@@ -35,6 +40,36 @@ pub struct PathfindingRequest {
     pub rolling_stock_supported_signaling_systems: Vec<String>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct RangeOffset {
```

Comment:
nit (feel free to disregard and resolve): more 🇬🇧 IMO
```suggestion
pub struct OffsetRange {
```